src/__tests__/lib/containers/LoadingScreen.test.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { BlockingLoader, BlockingLoaderProps } from '../../../lib/containers/LoadingScreen'
+import { mount } from 'enzyme'
+
+const mockCallback = jest.fn()
+
+const defaultProps: BlockingLoaderProps = {
+    show: true,
+    onCancel: undefined,
+    currentEntities: undefined,
+    totalEntities: undefined,
+}
+
+describe('basic functionality', () => {
```

Comment:
The test cases you have here are good. They don't need to change when you switch to RTL. Just code within the tests 🙂